data/json/items/book/misc.json

```diff
@@ -1033,6 +1033,34 @@
       {
         "id": "philosophy33",
         "text": "This is a copy of \"The Defence of Terrorism\" by Leon Trotsky.  Despite the title, it does not actually appear to be defending terrorism."
+      },
+      {
+        "id": "philosophy34",
+        "text": "This is a copy of \"Enquiry Concerning Political Justice\" by William Godwin.  This thick book is filled with antiquated phrases."
+      },
+      {
+        "id": "philosophy35",
+        "text": "This is a copy of \"The Abolition of Work and Other Essays\" by Bob Black.  It is likely that \"The Abolition of Work\" is the most famous essay in this book."
+      },
+      {
+        "id": "philosophy36",
+        "text": "This is a copy of \"What is Property?\" by Pierre-Joseph Proudhon.  It looks like that this book has a surprisingly long track record of owners."
```

Comment:
```suggestion
        "text": "This is a copy of \"What is Property?\" by Pierre-Joseph Proudhon.  It looks like this book has a surprisingly long track record of owners."
```

Reply:
That 'that' indeed looks unnecessary. Thank you for the review!